### DISCUSSION TITLE
ATSPM 4.3

### DISCUSSION THREAD

**Discussion by nicolassavard** [udotdevelopment/ATSPM#154] [Q&A] [answered]
Is there an Installation Manual for a new installation of the ATSPM 4.3 version? Thanks!

**Comment by runneals** [comment]
Also would be nice for a list of enhancements/major changes for 4.3 and other considerations for upgrading to 4.3. Was thinking something about the data architecture changing?

**Reply by mstanley-ave** [reply]
A list of changes for 4.3 can be found here: https://udottraffic.utah.gov/ATSPM/Home/About.
This lists the plans for 5.0 as well, which major changes in the backend and structure of the project.

**Comment by taylormd3** [comment]
All,

Attached is the installation manual for ATSPM V 4.3.

Regarding a list of the enhancements/major changes for V4.3, please go to:
https://udottraffic.utah.gov/atspm and visit the "ABOUT" page.

Sorry it took so long to get V4.3 out. However, a ton of time was spent
working out a lot of bugs and other things with it.

- Mark


*Mark Taylor, P.E.; PTOE | **Traffic Signal Operations Engineer*
*Utah Department of Transportation*
2060 South 2760 West | Salt Lake City, UT  84104
*Phone:*  *801-887-3714 <801-887-3714>* | ***@***.***

**Comment by taylormd3** [comment, marked as answer]
The PDF for the installation manual can be found here:

https://github.com/udotdevelopment/ATSPM/releases/tag/v4.3

-- Mark
